Modules/System/Cryptography Management/src/DESCryptoServiceProviderImpl.Codeunit.al

```diff
@@ -0,0 +1,94 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 1279 "DESCryptoServiceProvider Impl."
```

Comment:
This ID is reserved. Let's make it 1382

Reply:
@PredragMaricic, should I give "DESCryptoServiceProvider" then ID 1383?